user_docs/en/changes.t2t

```diff
@@ -21,23 +21,23 @@
 - Various padding and alignment issues in NVDA's dialogs have been resolved. (#6317, #5548, #6342, #6343, #6349)
 - The document formatting dialog has been adjusted so that the contents scrolls. (#6348)
 - Adjusted the layout of the symbols pronunciation dialog so the full width of the dialog is used for the symbols list. (#6101)
-- In browse mode in web browsers, the edit field (e and shift+e) and form field (f and shift+f) single letter navigation commands now move to read-only edit fields. (#4164)
+- In browse mode in web browsers, the edit field (e and shift+e) and form field (f and shift+f) single letter navigation commands can be used to move to read-only edit fields. (#4164)
 - In NVDA's Document Formatting settings, "Announce formatting changes after the cursor" has been renamed to "Report formatting changes after the cursor", as it affects braille as well as speech. (#6336)
 - Adjusted the appearance of the NVDA "Welcome dialog". (#6350)
 - NVDA dialog boxes now have their "ok" and "cancel" buttons aligned to the right of the dialog. (#6333)
-- Spin Controls are now used for numerical input fields. (#6099)
+- Spin Controls are now used for numeric input fields such as "Capital pitch change percentage" setting  in voice settings dialog. Enter the desired setting or use up or down arrow keys to change values. (#6099)
 
 
 == Bug Fixes ==
-- Fixed a rare issue when exiting NVDA while the speech viewer is open causing an error. (#5050)
+- Fixed a rare issue when exiting NVDA while the speech viewer is open which caused an error. (#5050)
 - Image maps now render as expected in browse mode in Mozilla Firefox. (#6051)
-- Fixed an issue causing a warning in the NVDA log file when caps-lock is toggled. (#6127)
-- While in the dictionary dialog, pressing the enter key now saves changes and exits. (#6206)
+- Fixed an issue causing a warning in the NVDA log file when caps lock is toggled. (#6127)

```

Comment:
You can remove "Fixed an issue causing a warning in the NVDA log file when caps lock is toggled. (#6127)". Thinking about it now, its not a user visible error and therefore does not need to be in the change log.